### PR TITLE
use say-as in GetPointOfInterestPrompt (#1720)

### LINKED ISSUE #1720: [Point of Interest] Set speak properties for address with "speak-as"/"interpret-as" attributes
## Project
**Name**: Point of Interest Skill
**Language**: C#

## Description

When listing out Points of Interest, the Skill says address names like "ave","st","blvd" literally rather than recognizing them as street names.

A speak property for a POI currently looks like:
`"speak": "Starbucks at 701 5th Ave #118 (at 4th & Cherry)"`

This should be updated to wrap SSML tags around an address like so:
`"speak": "Starbucks at <say-as interpret-as="address">701 5th Ave #118 (at 4th & Cherry)</say-as>"`

This would be useful as a method users can reuse from Microsoft.Bot.Solutions.
See documentation below for the SSML to use.

## To Reproduce

Test Point of Interest Skill on Direct Line Speech

## Expected behavior

Skill should speak abbreviated street names with full values

## Additional context

[Say-as documentation](https://docs.microsoft.com/en-us/cortana/skills/speech-synthesis-markup-language#say-as-element)

### PR DESCRIPTION
**Pull request by xieofxie**
## Purpose
*What is the context of this pull request? Why is it being done?*

Close #1720.

## Changes
*Are there any changes that need to be called out as significant or particularly difficult to grasp?*

*Include illustrative screenshots for context if applicable.*

Add say-as in GetPointOfInterestPrompt.

Since speak and voice elements are required in [cognitive service](https://docs.microsoft.com/en-us/azure/cognitive-services/speech-service/speech-synthesis-markup), the 'speak' has to be wrapped in html and one must pay attention to special characters in html.

Moreover, multiple language support is a little dirty in this scenario since we have to set related [voice](https://docs.microsoft.com/en-us/azure/cognitive-services/speech-service/language-support#text-to-speech) manually. 

## Tests
*Is this covered by existing tests or new ones? If no, why not?*

Tested in direct line speech with KonaClient.

## Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*

N/A.

### CODE REVIEW

File: skills/src/csharp/pointofinterestskill/pointofinterestskill/Dialogs/PointOfInterestDialogBase.cs [hunk 1]
```diff
@@ -411,7 +412,9 @@ protected PromptOptions GetPointOfInterestPrompt(string prompt, List<PointOfInte
             }
 
             options.Prompt = cards == null ? ResponseManager.GetResponse(prompt) : ResponseManager.GetCardResponse(prompt, cards);
-            options.Prompt.Speak = SpeechUtility.ListToSpeechReadyString(options.Prompt);
+
+            // TODO consider other languages
+            options.Prompt.Speak = $"<speak version='1.0' xmlns='https://www.w3.org/2001/10/synthesis' xml:lang='en-US'><voice name='en-US-Jessa24kRUS'>{SpeechUtility.ListToSpeechReadyString(options.Prompt, ReadPreference.Enumeration, 5)}</voice></speak>";
```
Comment (ryanisgrig) [inline skills/src/csharp/pointofinterestskill/pointofinterestskill/Dialogs/PointOfInterestDialogBase.cs]: Set the default en-us voice to "en-US-JessaNeural"

**Review by ryanisgrig — CHANGES_REQUESTED** [review]

File: skills/src/csharp/pointofinterestskill/pointofinterestskill/Dialogs/PointOfInterestDialogBase.cs (hunk 1 above)
Comment (ryanisgrig) [reply skills/src/csharp/pointofinterestskill/pointofinterestskill/Dialogs/PointOfInterestDialogBase.cs]: Also for other languages: https://docs.microsoft.com/en-us/azure/cognitive-services/speech-service/language-support

I would suggest using the neural for German, Italian, Chinese.

Comment (xieofxie) [reply skills/src/csharp/pointofinterestskill/pointofinterestskill/Dialogs/PointOfInterestDialogBase.cs]: Are these Female Neural voices the default voices for plain texts? I didn't find any document about this.

Comment (xieofxie) [reply skills/src/csharp/pointofinterestskill/pointofinterestskill/Dialogs/PointOfInterestDialogBase.cs]: Use dictionary SpeakDefaults for four neural voices.

Comment (ryanisgrig) [reply skills/src/csharp/pointofinterestskill/pointofinterestskill/Dialogs/PointOfInterestDialogBase.cs]: The Neural voices are better sounding and what we'd want to highlight should be used for a higher quality experience

**Review by ryanisgrig — APPROVED** [review]

**Pull request merged**